source/api.py

```diff
@@ -317,8 +317,15 @@ def getStatusBar():
 	if not location:
 		return None
 	left, top, width, height = location
-	bottom = top + height - 1
-	obj = getDesktopObject().objectFromPoint(left, bottom)
+	x=1
+	while x<12:
+		bottom = top + height - x
+		if left<0:
+			left=0

```

Comment:
For non maximized VS window, bottom=top+height-11 breaks if left is also
made to zero.
However if left is kept as it is and bottom=top+height-11, then status bar
is reported. But again here the solution fails for maximized window.

On Wed, Jan 13, 2016 at 12:14 PM, James Teh <notifications@github.com>
wrote:

> In source/api.py
> <https://github.com/nvaccess/nvda/pull/5620#discussion_r49556213>:
>
> > @@ -317,8 +317,15 @@ def getStatusBar():
> >  	if not location:
> >  		return None
> >  	left, top, width, height = location
> > -	bottom = top + height - 1
> > -	obj = getDesktopObject().objectFromPoint(left, bottom)
> > +	x=1
> > +	while x<12:
> > +		bottom = top + height - x
> > +		if left<0:
> > +			left=0
>
> And for non maximised, does bottom=top+height-11 work or break? I vaguely
> recall you saying it broke for non maximised.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nvaccess/nvda/pull/5620/files#r49556213>.
>



-- 
Regards
Manshul V Belani